wpilibj/src/main/java/edu/wpi/first/wpilibj/PWM.java

```diff
@@ -15,14 +15,21 @@
 /**
  * Class implements the PWM generation in the FPGA.
  *
- * <p>The values supplied as arguments for PWM outputs range from -1.0 to 1.0. They are mapped to
- * the hardware dependent values, in this case 0-2000 for the FPGA. Changes are immediately sent to
- * the FPGA, and the update occurs at the next FPGA cycle (5.005ms). There is no delay.
+ * <p>The values supplied as arguments for PWM Speed outputs range from -1.0 to 1.0 and PWM Position
+ * outputs range from 0.0 to 1.0 (for Servos). They are mapped to the hardware dependent values, in
+ * this case 0-2000 for the FPGA. Changes are immediately sent to the FPGA, and the update occurs at
+ * the next FPGA cycle (5.005ms). There is no delay.
  *
- * <p>As of revision 0.1.10 of the FPGA, the FPGA interprets the 0-2000 values as follows: - 2000 =
- * maximum pulse width - 1999 to 1001 = linear scaling from "full forward" to "center" - 1000 =
- * center value - 999 to 2 = linear scaling from "center" to "full reverse" - 1 = minimum pulse
- * width (currently .5ms) - 0 = disabled (i.e. PWM output is held low)
+ * <p>As of revision 0.1.10 of the FPGA, the FPGA interprets the 0-2000 values as follows:
+ *
+ * <ul>
+ *   <li>2000 = maximum pulse width
+ *   <li>1999 to 1001 = linear scaling from "full forward" to "center"
+ *   <li>1000 = center value
+ *   <li>999 to 2 = linear scaling from "center" to "full reverse"
+ *   <li>1 = minimum pulse width (currently .5ms)
+ *   <li>0 = disabled (i.e. PWM output is held low)
+ * </ul>
```

Comment:
The docs should mention somewhere in here that the "raw value" represents high time in microseconds